scripts/clean.js

```diff
@@ -5,7 +5,7 @@ const leaveIndex = (dir, basePath = 'src/pages/docs/') => {
 	// remove everything but leave dir's index.js/index.less
 	const fullPath = basePath + dir;
 	const entries = shelljs.ls('-d', fullPath + '/*');
-	const matchIndex = new RegExp(`${fullPath}\/index\.(js|less)`);
+	const matchIndex = new RegExp(`${fullPath}\/index\.(js|module\.less)`);
```

Comment:
Let's make the 'module' optional.  Something like `(js|(module\.)?less)`

Reply:
:+1: